### PR TITLE
sys-devel/prelink: add systemd timer

### PR DESCRIPTION
**Pull request by Tatsh**
Instead of cron job for those with systemd profiles.

### CODE REVIEW

File: sys-devel/prelink/prelink-99999999.ebuild [hunk 1]
```diff
@@ -52,6 +52,12 @@ src_install() {
 	exeinto /etc/cron.daily
 	newexe "${FILESDIR}"/prelink.cron prelink
 	newconfd "${FILESDIR}"/prelink.confd prelink
+
+	if use systemd; then
```
Comment (trofi) [inline sys-devel/prelink/prelink-99999999.ebuild:56]: Small files should be installed unconditionally without `use systemd` guard.

File: sys-devel/prelink/prelink-99999999.ebuild [hunk 2]
```diff
@@ -52,6 +52,12 @@ src_install() {
...
+		exeinto /usr/bin
+		newexe "${FILESDIR}"/prelink.cron systemd-prelink
```
Comment (trofi) [inline sys-devel/prelink/prelink-99999999.ebuild:58]: Renaming a fine is unfortunate here. I suggest using existing `/etc/cron.daily/prelink` file for timer unit and not introduce new script in `/usr/bin`. We can move it separately for both cron and time if need arises.

File: sys-devel/prelink/files/prelink.service [hunk 3]
```diff
@@ -0,0 +1,6 @@
+[Unit]
+Description=Prelink
```
Comment (trofi) [inline sys-devel/prelink/files/prelink.service:2]: Description should be more precise, like `Perform system-wide prelinking.`

File: sys-devel/prelink/files/prelink.service [hunk 4]
```diff
@@ -0,0 +1,6 @@
...
+
+[Service]
+Type=oneshot
+ExecStart=/usr/bin/systemd-prelink
```
Comment (trofi) [inline sys-devel/prelink/files/prelink.service:6]: I suggest using existing `/etc/cron.daily/prelink` path.

File: sys-devel/prelink/prelink-20151030-r1.ebuild [hunk 5]
```diff
@@ -56,6 +56,12 @@ src_install() {
 	exeinto /etc/cron.daily
 	newexe "${FILESDIR}"/prelink.cron prelink
 	newconfd "${FILESDIR}"/prelink.confd prelink
+
+	if use systemd; then
```
Comment (trofi) [inline sys-devel/prelink/prelink-20151030-r1.ebuild:60]: Same comments as in `-9999` ebuild

**Comment by Tatsh** [comment]
I made the changes suggested. Also added sign-off.

**Comment by trofi** [comment]
Pushed with a few minor routine changes:
- copied existing stable ebuild instead of moving
- dropped stabled keywords down to unstable for new revision

Thank you!

**Pull request closed without merging**